pkgs/development/python-modules/pyairports/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pyairports";
+  version = "2.1.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-PWCnJ/zk2oG5xjk+qK4LM9Z7N+zjRN/8hj90njrWK80=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
```

Comment:
please at least set `pythonImportsCheck`, and also set `doCheck = false` if there are no tests.